release.sh

```diff
@@ -99,7 +102,13 @@ commit_belongs_to_release_branch() {
       | grep -q -E '^origin/(main$|release/)'
 }
 
+prepare_snapshot() {
+    (./fmt.sh --rebuild-buf-image)
```

Comment:
Why do we need a subshell here?

Reply:
Because of the way `fmt.sh` is written (can be improved but wouldn't do it here) it `exit`s after the check to avoid also running the formatting logic.

Reply:
I fail to see the connection. That sounds like a good thing. 🤔

Reply:
`exit` from `fmt.sh` would fail to execute the rest of this script, like changing `LATEST`.

Reply:
I don't think it would:

```bash
$ echo "exit 1" > exit.sh
$ chmod +x exit.sh 
$ ./exit.sh 
$ 
```

Reply:
I am no `bash` expert, I'll re-test it and look for an explanation.

Reply:
Definitely not an expert either, but here's my explanation: any Bash command is calling a subprocess, and waiting for that subprocess to exit. A Bash script that does not have an explicit `exit` call still actually calls `exit` at the end of the file. That's true of any programming language: if you don't explicitly call exit, there's an implicit exit(0) at the end of your program.

There's no special case for when the program Bash starts is also a Bash process; it would be quite hard to use if it inlined every other shell script.

Note that none of that holds if you explicitly inline another Bash file, either through the `source` command or its alias `.`; i.e. `./fmt.sh` is a separate child process we wait on, whereas `. fmt.sh` is running within the current process.